proc_scripts/__init__.py

```diff
@@ -19,6 +22,15 @@
         'dBm2power',
         'power2dBm',
         'Vpp2power',
-        'center_CPMG_echo',
+        'find_echo_center',
+        'center_echo',
+        'recovery',
+        'decay',
+        'stft',
+        'isftf',
+        'simu_waves',
+        'simu_freq_sawtooth',
+        'integrate_limits',
+        'correl_align'
```

Comment:
I would sort this whole list alphabetically.

shift-V to highlight then `:sort`